packages/syft/src/syft/core/node/common/node_service/testing_services/smpc_executor_service.py

```diff
@@ -82,7 +82,7 @@ def process(
             result = StorableObject(
                 id=msg.id_at_location,
                 data=result,
-                read_permissions=store_object_self.read_permissions,
+                read_permissions={},
```

Comment:
I think we should use the same logic that normal operations do. If I call x.share() I should be able to call x.reconstruct() without a problem because the data originally came from me. The proposal above is more conservative, but at the cost of some mild functionality.